src/components/EventModal/Collect/index.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <TxModal
+    :is-open="isOpen"
+    :has-close-button="isShowCloseButton"
+    :header-text="headerText"
+    preset="collect"
+    @close="$emit('close')"
+    @handle-share="handleShare"
+    @go-portfolio="goToPortfolio"
+  >
+    <template #header-prepend>
+      <IconPrice />
+    </template>
+    <div
+      v-if="uiTxNFTStatus === 'completed'"
+      class="flex flex-col items-center justify-center mb-[12px]"
+    >
+      <Label
+        class="text-like-green font-600"
+        preset="h4"
+        :text="this.$t('tx_modal_status_complete_title')"
+      />
+      <Label
+        class="text-medium-gray mt-[12px]"
+        preset="h6"
+        :text="$t('tx_modal_status_complete_text_collect')"
+      />
+    </div>
+    <NFTPageOwning />
+    <section v-if="uiCollectMethodCallback">
+      <Label
+        class="text-like-green"
+        preset="h5"
+        align="center"
+        :text="$t('nft_collect_modal_subtitle_select_collect_method')"
+      />
+      <ul class="mt-[16px] flex flex-col gap-[16px]">
+        <li>
+          <EventModalCollectMethodButton
+            :title="$t('nft_collect_modal_method_crypto')"
+            :description="$t('nft_collect_modal_method_crypto_description')"
+            type="crypto"
+            :price="formattedLIKEPrice"
+            @click="uiCollectMethodCallback"
+          />
+        </li>
+        <li>
+          <EventModalCollectMethodButton
+            :title="$t('nft_collect_modal_method_stripe')"
+            :description="$t('nft_collect_modal_method_stripe_description')"
+            type="stripe"
+            :price="formattedFiatPrice"
+            @click="uiCollectMethodCallback"
```

Comment:
i think uiCollectMethodCallback is a strange name since it seems more like a trigger/handler function
Also strange to have ui logic depend on exist of this function or not